Neos.Neos/Classes/ViewHelpers/Link/NodeViewHelper.php

```diff
@@ -150,6 +146,12 @@ class NodeViewHelper extends AbstractTagBasedViewHelper
      */
     protected $throwableStorage;
 
+    /**
+     * @Flow\Inject
+     * @var NodeUriBuilderFactory
```

Comment:
we mix annotations and attributes in this class -_-

Reply:
yeah but its fluid 😎